.gitignore

```diff
@@ -11,3 +11,4 @@ nosetests.xml
 octodns.egg-info/
 output/
 tmp/
+/.vscode/
```

Comment:
Please remove this change.  octoDNS avoids including editor specific ignores here. I'd recommend looking at a "global" ignores file. 

```
 ross$ grep exclude ~/.gitconfig
  excludesfile = /Users/ross/.gitignores
```

```
(env)otter:datadog-dashboards ross$ cat ~/.gitignores
*.swo
*.swp
.DS_Store
.env/
tmp/
```

That gives you a way to ignore your editor's files without us having to add all of the possible options into octoDNS.

Reply:
Good call, thanks.